ci/ray_ci/macos/macos_ci_build.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -ex
+
+export CI="true"
+export PYTHON="3.8"
```

Comment:
should this be 3.9 now?

Reply:
some osx failed in 3.9 so we reverted it back to 3.8 for macos